internal/server/providers/config/config_e2e_test.go

```
@@ -0,0 +1,202 @@
+		t.Fatalf("rename: %v", err)
+	}
+}
+
+func gqlQuery(t *testing.T, ts *httptest.Server, doc string) []map[string]any {
+	t.Helper()
+	body, _ := json.Marshal(map[string]string{"query": doc})
+	resp, err := http.Post(ts.URL+"/graphql", "application/json", bytes.NewReader(body))
+	if err != nil {
+		t.Fatalf("post: %v", err)
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.1)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)